scripts/fdbshow

```diff
@@ -76,7 +78,10 @@ class FdbShow(object):
             if br_port_id not in self.if_br_oid_map:
                 continue
             port_id = self.if_br_oid_map[br_port_id]
-            if_name = self.if_oid_map[port_id]
+            if port_id.startswith("1"):
```

Comment:
I dont think this assumption can be hard-coded here. You can check if the port_id exists in `lag_if_oid_map`, if so fetch from that and if not, get from `if_oid_map`

Reply:
Hi prsunny,
It is a very good suggestion, I have modified and tested the modifaction. The new commit is 6f281af.
Thanks.

Reply:
I see that the commit Azure/sonic-py-swsssdk#51 will also fix the error. But I think a better modification is to provide API with a unique function such as API to get oid-map for interface/lag/bridge_port/vlan. 
I have push another pull request Azure/sonic-py-swsssdk#55 for this error fix.
Thanks.